tests/models/test_instrument.py

```diff
@@ -278,5 +288,136 @@ def _get_header_values_converters(cls):
     assert im_data.sum() > 0
     assert abs(np.mean(im_data) - 10) < 0.1  # random numbers with Poisson distribution around lambda=10
 
-
-# TODO: add more tests for e.g., loading FITS headers
\ No newline at end of file
+def test_demoim_search_notimplemented():
+    inst = DemoInstrument()
+    with pytest.raises( NotImplementedError ):
+        inst.find_origin_exposures()
+
+@pytest.fixture(scope='module')
+def decam_origin_exposures():
+    decam = DECam()
+    yield decam.find_origin_exposures( minmjd=60159.15625, maxmjd=60159.16667,
+                                       proposals='2023A-716082',
+                                       skip_exposures_in_database=False,
+                                       proc_type='instcal' )
+
```

Comment:
Maybe put this in conftest?

Reply:
My preference is for fixtures that are specific to a given test module to be in that module.  Otherwise, conftest.py gets huge.  conftest.py ought only to have things that are general, or that will be used in multiple different test modules.

Reply:
I can imagine using decam exposures in other places.